lib/mato/anchor_builder.rb

```diff
@@ -44,7 +44,13 @@ def make_anchor_id_suffix(text)
     end
 
     def make_anchor_id_prefix(text)
-      ERB::Util.url_encode(text.downcase.gsub(/[^\p{Word}\- ]/u, "").tr(" ", "-"))
+      prefix = ERB::Util.url_encode(text.downcase.gsub(/[^\p{Word}\- ]/u, "").tr(" ", "-"))
+
+      if prefix.empty?
+        "user-content"
```

Comment:
Assign `user-content` if anchor id is empty character.
`user-content` is used as anchor id prefix in GitHub, so use it.
<img width="775" alt="mato a pipeline-based markdown toolkit with commonmark er 2019-01-11 19-52-21" src="https://user-images.githubusercontent.com/1830471/51029758-8fbb9580-15da-11e9-8813-a4945cc3de2d.png">

Reply:
👍

Reply:
It's better if there's a comment as `# GitHub compatible`

Reply:
Added
32d7266